TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -0,0 +1,26 @@
+package page_objects;
+
+import com.logigear.control.common.imp.CheckBox;
+import com.logigear.control.common.imp.Link;
+import utils.common.Constants;
+
+public class DataProfilesPage {
+    private CheckBox inFrontCheckBox = new CheckBox("xpath=//table[@class='GridView']/tbody/tr/td[text()='%s']/preceding-sibling::td/input");
+    private Link dataProfileLink = new Link("xpath=//table[@class='GridView']/tbody/tr/td[text()='%s']");
+    private Link actionLink = new Link("xpath=//table[@class='GridView']/tbody/tr/td[text()='%s']/following-sibling::td[count(//table[@class='GridView']/tbody/tr/th)-2]");
```

Comment:
you are trying to use the table header but I don't see you specify the header anywhere.

Reply:
I have tried in another way. Please check again.

Reply:
Please check again. Structure of preset and non-preset are different